packages/salesforcedx-vscode-core/test/vscode-integration/commands/templates/forceFunctionCreate.test.ts

```diff
@@ -81,7 +75,7 @@ describe('Force Function Create', () => {
       })
     );
 
-    gatherStub = sandbox.stub(FunctionInfoGatherer.prototype, 'gather');
+    gatherStub = sandbox.stub(CompositeParametersGatherer.prototype, 'gather');
```

Comment:
`forceFunctionCreate` no longer uses `FunctionInfoGatherer`, so updating tests to account for that.